test/helpers/misc.rb

```diff
@@ -125,6 +125,13 @@ def skip_unless_minitest5_or_above
   skip 'This test requires MiniTest v5+'
 end
 
+def skip_unless_rails_gte(min_version)
```

Comment:
What does `gte` stand for?

Reply:
Oh, is it "greater than or equal"?

My mind went to LTS at first, which didn't seem right. Then GTSE, which also wouldn't work. It wasn't until I read the tests that I thought "greater than or equal". 

Could we call it `skip_unless_rails_version_at_least`? Or put a comment above the method explaining the abbreviation?

Reply:
It is indeed "greater than or equal". This one is so weird because it's really "greater than, or equal, or non-existent". Will  change it.

Reply:
Better name added with [aabe6c7](https://github.com/newrelic/newrelic-ruby-agent/pull/2155/commits/aabe6c7b50298004d7682881bc4b12f0fcab90bc)